src/rollup/types.d.ts

```diff
@@ -487,9 +492,10 @@ export interface SerializedTimings {
 }
 
 export interface OutputAsset {
+	type: 'asset';
 	code?: undefined;
 	fileName: string;
-	isAsset: true;
+	isAsset: true; // deprecated
```

Comment:
make a getter. with [deprecation message](https://github.com/rollup/rollup/pull/3080#issuecomment-524608261)

Check if setting to isAsset changes the behavior of the program. if yes only getter is enouh if not setter needs to be implemented.

Reply:
```suggestion
  /** @deprecated Use type === 'asset' instead. */
	isAsset: true;
```

typescript tooling can detect usage of deprecated properties using @deprecated 

This way most developers could get a warning at compiler time!